.clang-format

```diff
@@ -0,0 +1,66 @@
+# Generated from CLion C/C++ Code Style settings
```

Comment:
Did you mean to commit this?
We will have a unified clang-format settings, so it's not a bad idea IMO.

Reply:
yep, but feel free to remove the comment/modify accordingly. strong in favour of clang-formatting the whole code base

Reply:
Ah, I meant the whole file. A common clang-format config is a good idea then.